### PR TITLE
feat: article bottom banner

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 文章頁 bottom banner](https://app.asana.com/0/1203699264568808/1205329976610294/f)

# Notice

# Dependency
[npm-packages#447](https://github.com/twreporter/twreporter-npm-packages/pull/447)

### CODE REVIEW

File: src/components/membership-promo/article-banner.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        iconComponent={<Cross releaseBranch={releaseBranch} />}
+        theme={IconButton.THEME.normal}
+        onClick={closePromo}
+      />
+      <FlexRow>
+        <PencilImg src={imageUrl} alt="有你才有報導者" />
+        <TextBox>
+          <Title text="有你才有報導者" />
+          <Description text="這篇文章的完成有賴讀者的贊助支持，我們以非營利模式運作，" />
+          <Description text="邀請你加入3種支持方案，讓報導者能夠走更長遠的路。" />
```
Comment (Aylie-Chou) [inline src/components/membership-promo/article-banner.js:96]: 盤古之白？

File: src/components/membership-promo/article-banner.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      <FlexGroup>
+        <Title text="有你才有報導者" />
+        <CloseButton
+          iconComponent={<Cross releaseBranch={releaseBranch} />}
+          theme={IconButton.THEME.normal}
+          onClick={closePromo}
+        />
+      </FlexGroup>
+      <DescriptionBox>
+        <Description text="這篇文章有賴讀者的贊助完成，我們以非營利模式運作，邀請你加入3種支持方案，讓我們能走更長遠的路。" />
```
Comment (Aylie-Chou) [inline src/components/membership-promo/article-banner.js:149]: 盤古之白？

File: src/components/membership-promo/article-banner.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+  )
+}
+
+const Box = styled.div`
+  visibility: ${props => (props.show ? 'visible' : 'hidden')};
+  ${props => (props.show ? '' : 'transition: visibility 0.5s linear 0.5s;')}
+`
+const ArticleBanner = ({ isExpanded = false, isAuthed = false }) => {
+  const [isShowPromo, setIsShowPromo] = useState(false)
+  const [isOpened, setIsOpened] = useState(false)
```
Comment (Aylie-Chou) [inline src/components/membership-promo/article-banner.js:162]: Question: what's the diffrenece between `isShowPromo` & `isOpened` state ?

File: src/components/membership-promo/article-banner.js (hunk 1 above)
Comment (liruchen32) [reply src/components/membership-promo/article-banner.js]: fixed

File: src/components/membership-promo/article-banner.js (hunk 2 above)
Comment (liruchen32) [reply src/components/membership-promo/article-banner.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]

**Pull request merged**